app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<%# path = app/views/expenses/new.html.erb %>
+<div class="categories-index">
+  <div class="header">
+    <span><a href="<%= categories_path %>" class="back-arrow">←</a></span>
+    <span>ADD EXPENSE</span>
+    <span><a href="<%= categories_path %>" class="back-arrow">&#128269;</a></span>
+  </div>
```

Comment:
- [ ] On "Add a new transaction" page when the user clicks on the "Back" button (<), the user should redirects to the transactions page for that category but currently user redirects to the homepage(category page). Please fix this issue.

![image](https://github.com/noelfoka/capstone_budget_app/assets/83803414/3758df61-8fd6-4e59-845a-c039a91380ad)